### DISCUSSION TITLE
Vortex-Radioss v1.0.0

### DISCUSSION THREAD

**Discussion by Vortex-CAE** [Vortex-CAE/Vortex-Radioss#14] [Announcements]
First public release of Vortex-Radioss

Geometry conversion 1D, 2D, 3D

2D:
- Effective plastic strain
- Stress tensor
- Internal energy density
- Thickness
- Part masses

<hr /><em>This discussion was created from the release <a href='https://github.com/Vortex-CAE/Vortex-Radioss/releases/tag/'>Vortex-Radioss v1.0.0</a>.</em>

**Comment by ainurritegui** [comment]
I am trying to run this command, but I cannot. How do I have to run it??
I have all the dowloaded files and I my OpenRadiuss results in the same folder.
I run this folder from the cmd. 
And then try to run this.. but I think I am missing something in the middle...

# Load the module
from vortex_radioss.animtod3plot.RadiossReader import RadiossReader

# Generate animation object
rr = RadiossReader("fileA001")

### To display high level information about the arrays
# Return header dictionary
rr.raw_header

Do I have to put any path variables? Can you help me?
Thank you very much!

**Reply by Vortex-CAE** [reply]
Hi @ainurritegui,

You will need to make sure that the library is in your python interpreter's path. You will also need to make sure that the folder names matches the import statement. Probably best to put it in the same directory as where the numpy or other common packages exists.

We do plan on making this package pip installable in future.

Kind regards, 
David
